bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableDataGrpcClient.java

```diff
@@ -408,6 +413,7 @@ private Metadata createMetadata(String tableName) {
   private ResultScanner<Row> streamRows(ReadRowsRequest request) {
     expandPoolIfNecessary(this.bigtableOptions.getChannelCount());
 
+    Timer.Context timeContext = readRowsAsync.getRpcMetrics().timeRpc();
```

Comment:
no `r`...  that's a typo.  I'll look through our variable names and make them all `timerContext`